Makefile

```
@@ -1,6 +1,9 @@
-# minimal Makefile to run Python unit tests
-# for flux-accounting
+FLUX_INSTALL_PREFIX ?= /usr
+
+install:
+	${FLUX_INSTALL_PREFIX}bin/flux python -m pip install -r requirements.txt --user
+	${FLUX_INSTALL_PREFIX}bin/flux python -m pip install --install-option="--prefix=${FLUX_INSTALL_PREFIX}" \
+	--install-option="--install-scripts=${FLUX_INSTALL_PREFIX}libexec/flux/cmd/" . --upgrade
```

Generally looks good.  Just need some extra slashes in case the user doesn't put a trailing slash at the end of `FLUX_INSTALL_PREFIX`.

```suggestion
	${FLUX_INSTALL_PREFIX}/bin/flux python -m pip install -r requirements.txt --user
	${FLUX_INSTALL_PREFIX}/bin/flux python -m pip install --install-option="--prefix=${FLUX_INSTALL_PREFIX}" \
	--install-option="--install-scripts=${FLUX_INSTALL_PREFIX}/libexec/flux/cmd/" . --upgrade
```